dap-java.el

```diff
@@ -108,7 +108,7 @@ If the port is taken, DAP will try the next port."
   (lsp-interface
    (java:MainClass (:mainClass :projectName))))
 
-(defun dap-java-test-class ()
+(defun dap-java-test-class (&optional no-signal?)
```

Comment:
seems like this change is not needed?

Reply:
Ahh yes, I just added it for symmetry with the dap-java-test-method-at-point (in case someone wants to use that function similarly). Would you like me to remove it?

Reply:
@yyoncho any opinion on the above? Would be nice to get this merged.